elements/rh-navigation-primary/rh-navigation-primary-item.css

```diff
@@ -229,29 +223,36 @@
           cursor: pointer;
 
           &:before {
-            --_mask: conic-gradient(transparent 0 0);
-
             content: '';
             position: absolute;
             inset: -1px;
+            z-index: -1;
+            background: transparent;
             border-radius: inherit;
-            padding: var(--rh-border-width-sm, 1px);
-            background: var(--_active-item-color);
-            mask: var(--_mask) content-box exclude, var(--_mask);
-            z-index: 1;
           }
 
-          &:active {
+          &:is(:hover, :focus-visible) {
+            background-color:
+              light-dark(var(--rh-color-surface-lighter, #f2f2f2),
+                var(--rh-color-surface-darker, #1f1f1f));
+
             &:before {
-              padding: var(--rh-border-width-md, 2px);
+              border: var(--rh-border-width-sm, 1px) solid transparent;
             }
           }
 
-          &:is(:hover, :focus-visible) {
-            background-color: var(--_background-color);
+          &:is(:active, :active:hover) {
+            &:before {
+              border: var(--rh-border-width-md, 2px) solid transparent;
+            }
+          }
 
+          &:is(:active, :active:hover, :hover, :focus-visible) {
             &:before {
-              --_mask: conic-gradient(#000000 0 0);
+              background:
```

Comment:
might be worth naming this as a --_var, just for consistency with --_linear-gradient